settings.gradle

```diff
@@ -30,6 +31,24 @@ gradleEnterprise {
     }
 }
 
+ext.isCiServer = System.getenv().containsKey("CI")
+
+buildCache {
+    local {
```

Comment:
@cpdeethree can we enable this? If we are afraid of this edge case, I think education is a better path here.